<ui version="4.0" >
 <class>ShareFilesDialog</class>
 <widget class="QDialog" name="ShareFilesDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>557</width>
    <height>315</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Shared Folder Manager</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" >
   <item row="0" column="0" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Shared Folder Manager:</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" >
      <item row="1" column="0" >
       <widget class="QPushButton" name="addfile_Btn" >
        <property name="text" >
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QPushButton" name="remfile_Btn" >
        <property name="text" >
         <string>Remove</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2" >
       <spacer name="horizontalSpacer" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>170</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="3" >
       <widget class="QPushButton" name="shareOk" >
        <property name="text" >
         <string>Ok</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4" >
       <widget class="QPushButton" name="cancel_pushButton" >
        <property name="text" >
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="5" >
       <widget class="QListWidget" name="DownloadList" />
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancel_pushButton</sender>
   <signal>clicked()</signal>
   <receiver>ShareFilesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>499</x>
     <y>283</y>
    </hint>
    <hint type="destinationlabel" >
     <x>278</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
